src/test/java/hw2/TestBase.java

```diff
@@ -0,0 +1,82 @@
+package hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.asserts.SoftAssert;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TestBase {
+
+    private static final String DEFAULT_URL = "https://jdi-testing.github.io/jdi-light/index.html";
+    private static final String URL = System.getProperty("url", DEFAULT_URL);
+
+    private static final String DEFAULT_LOGIN = "Roman";
+    private static final String DEFAULT_PASSWORD = "Jdi1234";
+    private static final String DEFAULT_USERNAME = "ROMAN IOVLEV";
+    private static final String LOGIN = System.getProperty("login", DEFAULT_LOGIN);
+    private static final String PASSWORD = System.getProperty("password", DEFAULT_PASSWORD);
+    private static final String USERNAME = System.getProperty("user", DEFAULT_USERNAME);
+
+    private static final String EXPECTED_HOME_PAGE_TITLE = "Home Page";
+
+    private static final String LOGIN_DROPDOWN_CLASS_SELECTOR = "uui-profile-menu";
```

Comment:
Do not create constants for locators, use in directly in method calls, remove all of them

Reply:
done